circle.yml

```diff
@@ -1,12 +1,9 @@
 machine:
   node:
     version: 4.2.1
-  ruby:
-    version: 2.2.3
```

Comment:
No more Ruby needed on our CI since we get our changelog by [github changelog api](https://github.com/levithomason/github-changelog-api/) now. :D